src/test/ui/duplicate/duplicate-check-macro-exports.stderr

```diff
@@ -1,18 +1,15 @@
-error[E0255]: the name `panic` is defined multiple times
-  --> $DIR/duplicate-check-macro-exports.rs:4:1
+error[E0252]: the name `panic` is defined multiple times
+  --> $DIR/duplicate-check-macro-exports.rs:3:1
    |
-LL | pub use std::panic;
-   |         ---------- previous import of the macro `panic` here
-...
-LL | macro_rules! panic { () => {} }
-   | ^^^^^^^^^^^^^^^^^^ `panic` redefined here
+LL |   pub use std::panic;
+   |           ---------- previous import of the macro `panic` here
+LL |
+LL | / #[macro_export]
+LL | | macro_rules! panic { () => {} }
+   | |_______________________________^ `panic` reimported here
```

Comment:
"reimported" is very surprising to a user who does not know that `macro_export` acts like an import.

Reply:
I've updated the PR to avoid changes in diagnostics.